website/content/docs/connect/observability/ui-visualization.mdx

```diff
@@ -497,6 +581,23 @@ ui_config {
 }
 ```
 
+</Tab>
+<Tab heading="JSON">
+
+```json
+{
+  "ui_config": {
+    "enabled": true,
+    "metrics_provider": "example-provider",
+    "metrics_provide_files": ["/usr/local/bin/example-metrics-provider.js"],
+    "metrics_provider_options_json": "{\"foo\":\"bar\"}"
```

Comment:
It might be helpful to add an example of how an operator can JSON-encode the value for `metrics_provider_options_json`. For example, https://www.consul.io/docs/connect/proxies/envoy#configuration-formatting.

Reply:
nice! what would be the best approach here @blake, should we link to the explanation we already have, or duplicate it within this document?

Reply:
I think it'd be nice to have a central place on the docs where we show how to JSON encode values like this. @trujillo-adam Do you have an opinion on where we could centralize this info?

Reply:
@krastin instead of `<Tabs>` we should be using the < CodeTabs> component instead. I created a [video](https://hashicorp.atlassian.net/wiki/spaces/NET/pages/2259910751/How+to+contribute+to+Consul+documentation#Web-Components) that walks through how to use this component.

Reply:
@karl-cardenas-coding apologies, i assumed i should follow the present structure per page. would you like me to refactor all the `<Tabs>` sections into `<CodeTabs>` in the same PR? i think it would makes sense in this case

Reply:
@krastin Yes sir 😄